easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.2-foss-2016a-R-3.2.3.eb

```diff
@@ -13,136 +13,134 @@ toolchain = {'name': 'foss', 'version': '2016a'}
 exts_defaultclass = 'RPackage'
 exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
 
-rver = '3.2.3'
-versionsuffix = '-R-%s' % rver
-dependencies = [('R', rver)]
+dependencies = [('R', '3.2.3')]
 
-name_tmpl = '%(name)s_%(version)s.tar.gz'
-ext_options = {
+local_name_tmpl = '%(name)s_%(version)s.tar.gz'
+local_ext_options = {
     'source_urls': [
         'http://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
         'http://cran.r-project.org/src/contrib/',  # current version of packages
         'http://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
     ],
-    'source_tmpl': name_tmpl,
+    'source_tmpl': local_name_tmpl,
 }
-bioconductor_options = {
+local_bioconductor_options = {
```

Comment:
Is `exts_default_options` viable here?

Reply:
Yes, but I'd prefer not making that change since it's a lot harder to review (and there's very little benefit for an old easyconfig like this).